test/util/test_app.go

```diff
@@ -178,7 +187,27 @@ func SetupDeterministicGenesisState(testApp *app.App, pubKeys []cryptotypes.PubK
 func NewTestAppWithGenesisSet(cparams *tmproto.ConsensusParams, genAccounts ...string) (*app.App, *tmtypes.ValidatorSet, keyring.Keyring) {
 	testApp := NewTestApp()
 	genesisState, valSet, kr := GenesisStateWithSingleValidator(testApp, genAccounts...)
+	testApp = InitialiseTestAppWithGenesis(testApp, cparams, genesisState)
+	return testApp, valSet, kr
+}
 
+// NewTestAppWithGenesisSetAndMaxSquareSize initializes a new app with genesis accounts and a specific max square size
+// and returns the testApp, validator set and keyring.
+func NewTestAppWithGenesisSetAndMaxSquareSize(cparams *tmproto.ConsensusParams, maxSquareSize int, genAccounts ...string) (*app.App, *tmtypes.ValidatorSet, keyring.Keyring) {
+	testApp := NewTestApp()
+	genesisState, valSet, kr := GenesisStateWithSingleValidator(testApp, genAccounts...)
+
+	// hacky way of changing the gov max square size without changing the consts
```

Comment:
@cmwaters I changed the max square size in here in this hacky way. If you think it's fine, I can make this PR ready for review.

Reply:
There is a `genesis` package which looks like could do this functionality for you

Reply:
Can you link me please to which part can do this for me?

Reply:
if we just want to change the square size, we can use https://github.com/celestiaorg/celestia-app/blob/3ea0ca5bd589d7212dbd429eaca8365ff2f35dc9/test/util/genesis/modifier.go#L24-L32

Reply:
https://github.com/celestiaorg/celestia-app/issues/3971